### DISCUSSION TITLE
External data storage block

### DISCUSSION THREAD

**Discussion by promitheas-nikou** [cc-tweaked/CC-Tweaked#1122] [General]
Computers can store files internally (up to 1MB by default I think) and the user can also place adjacent disk drives to store more data. But floppy drives can only store little data (500KB by default, yes I know it can be changed in the config). Shouldn't there also be another block that would store more data, or at least something that can hold many floppy disks (maybe a RAID)?

**Comment by xAnavrins** [comment]
You can also use computer/pocket comp as floppies, essentially giving you 1MB instead of the default 125kB of floppies.

**Reply by SquidDev** [reply]
There's also config options if you want to bump the disk/computer size.

**Reply by promitheas-nikou** [reply]
Yes, I know it can be changed it (I even wrote that in my post...), but I just think it would be nice if there were some clean and compact way to store a lot of data together
